firstvoices/backend/tasks/build_mtd_export_format_tasks.py

```diff
@@ -134,3 +138,50 @@ def build_index_and_calculate_scores(site_or_site_slug: str | Site, *args, **kwa
     MTDExportFormat.objects.filter(site=site).exclude(id=new_result.id).delete()
 
     return preview
+
+
+@shared_task(bind=True)
+def check_sites_for_mtd_sync(self):
+    try:
+        LOGGER.info("Starting MTD Sync check.")
+        self.state = "STARTED"
+        sites = Site.objects.all()
```

Comment:
We can filter this to only public sites

Reply:
Which will be good, since we can no longer filter the dictionary entries based on visibility. (We could catch all the currently public ones that changed, but we would miss any formerly public ones that got made private.)